### PR TITLE
Paul/check gfzrnx execution

### PR DESCRIPTION
**Pull request by plutonheaven**
Adds Windows binary for gfzrnx
Updates Linux and MacOS binaries
Rewrites helpers.repair_with_gfzrnx using looser execution success criteria
Adds tests to verify the execution of gfzrnx

This should solve the Windows issue in #53.

### CODE REVIEW

**Comment by plutonheaven** [comment]
@jtec, I wonder if there is a better way to use the right binary depending on the OS. Here, testing all 3 binaries (and knowing that at least 2 of them will fail) complexifies a bit the code.

File: src/prx/helpers.py [hunk 1]
```diff
@@ -126,18 +124,34 @@ def rinex_header_time_string_2_timestamp_ns(time_string: str) -> pd.Timestamp:
...
+            result = subprocess.run(
+                command,
+                capture_output=True,
+                shell=True,
+            )
+            if result.returncode == 0:
+                log.info(f"Ran gfzrnx file repair on {file}")
+            else:
+                log.info(f"gfzrnx file repair run failed: {result}")
+        except OSError:
```
Comment (jtec) [inline src/prx/helpers.py:150]: Shouldn't we fail early when this happens, as other stuff can break downstream as a consequence of this step failing?

**Comment by jtec** [comment]
> @jtec, I wonder if there is a better way to use the right binary depending on the OS. Here, testing all 3 binaries (and knowing that at least 2 of them will fail) complexifies a bit the code.

@plutonheaven Good call - you could use `platform.system()` and then look up the binary to use from a dicrionary.

File: src/prx/helpers.py (hunk 1 above)
Comment (plutonheaven) [reply src/prx/helpers.py]: Ok, I returned to strict execution check. Any failed execution of `gfzrnx` will stop the program.

**Comment by plutonheaven** [comment]
@jtec, I did the suggested actions:
- choose binary according to `platform.system()`
- follow strict execution check of `gfzrnx`

Everything works on both Windows and Linux! Let's check on MacOS.

**Comment by jtec** [comment]
> @jtec, I did the suggested actions:
> 
>     * choose binary according to `platform.system()`
> 
>     * follow strict execution check of `gfzrnx`
> 
> 
> Everything works on both Windows and Linux! Let's check on MacOS.

Ran https://github.com/jtec/prx/blob/7d4fcd2e977146152a534c2eac2034daa28909ad/src/prx/test/test_helpers.py#L258 on Mac OS, no problem! Thanks for taking this on!

**Review by jtec — APPROVED** [review]

File: src/prx/test/test_helpers.py [hunk 2]
```diff
@@ -156,3 +203,69 @@ def test_sagnac_effect():
     # millimeter accuracy should be sufficient
     tolerance = 1e-3
     assert np.max(np.abs(sagnac_effect_computed - sagnac_effect_reference)) < tolerance
+
+
+def test_gfzrnx_execution_on_obs_file(input_for_test):
```
Comment (jtec) [inline src/prx/test/test_helpers.py:208]: Thanks for adding these tests!

**Pull request merged**